book/accessibility.md

```diff
@@ -2370,21 +2441,6 @@ sufficient contrast between the text and whatever is behind it.
 
 [forced-colors]: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors
 
-*`:hover` pseudo-class*: There is a `:hover` pseudo-class that
```

Comment:
Why remove this?

Reply:
Oh! I moved it to Chapter 16 instead. The proposed solution here is render, invalidate hover, rerender. I think that would take a long time on this chapter but will be quite fast in most scenarios after invalidation is added, so I figured it'd be a better exercise in Chapter 16.